contracts/koh/contracts/KingAutomaton.sol

```diff
@@ -178,6 +185,29 @@ contract KingAutomaton {
     }
 
     payForGas(id, 1);
+    return id;
+  }
+
+  function createProposal(address payable contributor, string memory title, string memory documents_link,
+      bytes memory documents_hash, uint256 contest_period, int256 approval_percentage, int256 contest_percentage)
```

Comment:
``uint256 contest_period, int256 approval_percentage, int256 contest_percentage``
let's make these global parameters for now